tests/index.py

```
@@ -0,0 +1,450 @@
+
+        Returns:
+            JSON response with auto-completion suggestions or an error message.
+        """
+
+        # Get JSON data from the request
+        data = request.get_json()
+
+        # Validate that 'repo' is provided
+        repo = data.get('repo')
+        if repo is None:
+            return jsonify({'status': f'Missing mandatory parameter "repo"'}), 400
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)